hazelcast/src/main/java/com/hazelcast/client/impl/ClientEndpointImpl.java

```diff
@@ -61,6 +61,8 @@
     private volatile boolean authenticated;
     private String clientVersion;
     private volatile String stats;
+    private long authenticationCorrelationId;
```

Comment:
This should have been removed. It looks like it got back because of a rebase error